AllReadyApp/Web-App/AllReady/Services/AttachmentService.cs

```diff
@@ -7,13 +7,16 @@
 using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.Blob;
 using System.Linq;
+using System.Collections.Generic;
 
 namespace AllReady.Services
 {
     public class AttachmentService : IAttachmentService
     {
         private const string ContainerName = "attachments";
-        public static string[] AllowedExtensions = new string[] { ".jpg", ".gif", ".bmp", ".pdf" };
+        private const int MaxAttachmentBytes = 2 * 1024 * 1024; // 2MB
+        private static readonly IList<string> AllowedExtensions = new List<string> { ".png", ".jpg", ".doc", ".docx", ".xls", ".xlsx", ".pdf" };
```

Comment:
please delete this and any functionality that uses it. We don't need to restrict what the user can attach, the Issue has not mention of that